docs/DeveloperGuide.md

```diff
@@ -8,22 +8,107 @@
 
 {Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
```

Comment:
Better to re-iterate the value proposition of your app at the beginning of the design and implementation section.